lib/views/chatroom/views/chatroom_participants_page.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:group_chat_example/utils/ui_utils.dart';
+import '../../../widgets/back_button.dart' as BB;
+
+class ChatroomParticipantsPage extends StatefulWidget {
+  const ChatroomParticipantsPage({super.key});
+
+  @override
+  State<ChatroomParticipantsPage> createState() =>
+      _ChatroomParticipantsPageState();
+}
+
+class _ChatroomParticipantsPageState extends State<ChatroomParticipantsPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        children: [
+          const SizedBox(height: 72),
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 24),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                const BB.BackButton(),
+                Text(
+                  "Chatroom Participants",
+                  style: GoogleFonts.montserrat(
+                    fontSize: 18,
+                    fontWeight: FontWeight.w600,
+                  ),
+                ),
+                const SizedBox(width: 32),
+              ],
+            ),
+          ),
+          const SizedBox(height: 8),
+          Text(
+            "10 participants",
+            style: GoogleFonts.roboto(
+              fontWeight: FontWeight.w300,
+            ),
+          ),
+          Expanded(child: _buildParticipantsList()),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildParticipantsList() {
```

Comment:
Please make sure to use class widgets instead of functional widgets where ever possible.

Reply:
This is only because I have not finalised the data layer structure, all of this is just temporary to structure the UI. Once I get the data layer in place, all of this will be coming from the BLoC anyway.